packages/react/README.md

```diff
@@ -169,7 +169,7 @@ Mute the assistant audio
 
 Unmute the assistant audio
 
-#### `sendSessionSettings: (text: string) => void`
+#### `sendSessionSettings: (message: SessionSettings) => void`
```

Comment:
i don't believe we document what `SessionSettings` looks like anywhere in the readme? that seems important to add too

Reply:
Good point! What do you think of linking to the type in the TypeScript sdk?

something like:

#### sendSessionSettings: (message: [SessionSettings](https://github.com/HumeAI/hume-typescript-sdk/blob/ac89e41e45a925f9861eb6d5a1335ab51d5a1c94/src/api/resources/empathicVoice/types/SessionSettings.ts)) => void

Reply:
I could do this for the other types we do not mention in the README as well